src/error.rs

```diff
@@ -111,6 +111,9 @@ pub enum Error {
         container_name: String,
         violation: String,
     },
+
+    #[error("The duration {duration:?} is too long to process")]
+    DurationTooLong { duration: Duration },
```

Comment:
Is too long for what? Where does the duration come from? What is the limit? Where's the source error?

Reply:
If we expect the downstream operator to provide this context, make a separate error type instead of adding even more crap to the overbloated corpse that is `OperatorError`.

Reply:
Please have another look.